apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -288,28 +288,49 @@ defmodule SiteWeb.CustomerSupportController do
     end
   end
 
-  @spec validate_incident_date_time(map) :: DateTime.t()
-  defp validate_incident_date_time(incident_date_time) do
-    now = Util.now() |> Util.to_local_time()
-
-    parsed_date_time =
-      case Util.parse(incident_date_time) do
-        {:error, :invalid_date} ->
-          now
-
-        parsed_dt ->
-          parsed_dt
-      end
-
-    local_parsed_date_time = Util.to_local_time(parsed_date_time)
-
-    # if date and time is in the future, set it to now
-    # otherwise leave as it is
+  @spec validate_incident_date_time(map) :: map
+  def validate_incident_date_time(incident_date_time) do
+    local_now = Util.now() |> Util.to_local_time()
```

Comment:
It might make it easier to test if the `now` can also be an argument to the function: `def validate_incident_date_time(incident_date_time, now \\ Util.now()) do`